mon-pix/app/pods/components/module/details/styles.scss

```
@@ -1,21 +1,131 @@
-.module {
+.module-details {
   max-width: 960px;
   margin: 0 auto;
 
-  &__title {
-    @extend %pix-title-m;
+  &__text {
+    @extend %pix-body-m;
 
-    margin: var(--pix-spacing-12x) var(--pix-spacing-2x);
-    text-align: center;
+    padding: var(--pix-spacing-2x) 0;
+    color: var(--pix-neutral-900);
+    font-weight: var(--pix-font-normal);
   }
 
   &__content {
-    --modulix-grains-gap: 56px;
-
     display: flex;
     flex-direction: column;
-    gap: var(--modulix-grains-gap);
+    gap: var(--pix-spacing-6x);
+    align-items: flex-start;
+    padding: var(--pix-spacing-8x) var(--pix-spacing-6x);
+    background-color: var(--pix-primary-10);
+  }
+
+  &__image {
+    width: 100%;
+    background-color: var(--pix-error-100);
+
+    img {
+      display: block;
+      margin: 0 auto;
+    }
+  }
+
+  &__infos {
+    display: flex;
+    flex-direction: row;
+    gap: var(--pix-spacing-3x);
     align-items: center;
-    justify-content: center;
+    align-self: stretch;
+    justify-content: space-around;
+    padding: var(--pix-spacing-2x) var(--pix-spacing-8x);
+    background-color: var(--pix-primary-100);
+    border-radius: 0 0 16px 16px;
+  }
+
+  &-infos {
+    &__category {
+      @extend %pix-body-l;
+
+      color: var(--pix-neutral-900);
+      font-weight: var(--pix-font-medium);
+    }
+
+    &__duration {
+      p::before {
+        content: '≈'
+      }
+    }
+  }
+
+  &__objectives {
+    padding: var(--pix-spacing-8x) var(--pix-spacing-6x);
+
+    ol {
+      padding: 0;
+      list-style: none;
+      counter-reset: item;
+    }
+
+    li {
+      @extend %pix-body-m;
+
+      display: flex;
+      align-items: center;
+      margin-bottom: 16px;
+      color: var(--pix-neutral-900);
+      font-weight: var(--pix-font-normal);
+      counter-increment: item;
+    }
+
+    li::before {
+      @extend %pix-body-m;
+
+      display: inline-block;
+      min-width: 32px;
+      height: 32px;
+      margin-right: 10px;
+      color: var(--pix-primary-700);
+      line-height: 32px;
+      text-align: center;
+      background: var(--pix-primary-100);
+      border-radius: 100%;
+      content: counter(item);
+    }
   }
+
+  &__explanation {
+    padding: 0 var(--pix-spacing-6x);
+  }
+}
+
+.module-details-content {
+  &__title {
+    @extend %pix-title-m;
+
+    text-align: left;
+  }
+
+  &__description {
+    @extend %pix-body-m;
+
+    color: var(--pix-neutral-900);
+  }
+}
+
+.infos__title {
+  @extend %pix-body-s;
+
+  color: var(--pix-neutral-800);
+  font-weight: var(--pix-font-normal);
```

C'est sensé être la valeur par défaut

```suggestion
  font-weight: var(--pix-font-normal);
```

---

fixé